### ISSUE TITLE
How about add "handlebars.enabled" property?

### ISSUE THREAD

**Issue by uuidcode** [allegro/handlebars-spring-boot-starter#17]
I'm writing core library that supports multiple ViewResolver.
And I like to choose single ViewResolver at application.properties.
So how about add "handlebars.enabled" property below code like?

```
@ConditionalOnProperty(name = "handlebars.enabled", matchIfMissing = true)
@Configuration
@EnableConfigurationProperties(HandlebarsProperties.class)
@ConditionalOnWebApplication
public class HandlebarsAutoConfiguration
```

**Comment by plesiecki** [comment]
Feel free to open a pull request.

**Comment by plesiecki** [comment]
fixed in 0.2.12
